### DISCUSSION TITLE
Windows <> Mac: Map Windows Key to Left or Right Command Key?

### DISCUSSION THREAD

**Discussion by throwawayburner69** [deskflow/deskflow#7857] [Q&A]
Server: Windows 11
Client: MacOS 18.1
Deskflow version: Continuous (11/1/24)

---

Is there a way to map the Windows key to the left or right Command key? If I/you enable the MacOS on-screen keyboard, it shows a single Windows key press as a simultaneous keypress of both the left and right Command keys.

Scenario/use case: "type to Siri" allows for a double Command keypress (read: pressing twice - not both) to trigger, and this doesn't work as it's looking for a double key press of an individual Command key.

Thanks!

**Comment by nbolton** [comment]
Try swapping the keys in the macOS keyboard settings. For the Deskflow client, you can also swap the keys in a similar way under server settings (on the server).

**Reply by throwawayburner69** [reply]
Thanks - I guess I'm not looking to swap the keys on the Mac side - just be able to specify right vs left command. On the Windows server settings, I can remap super/meta keys to others (only Ctrl, Alt, etc.), but I'm not seeing an option to force/bind it to the left vs right Command/meta key on Mac.

**Reply by nbolton** [reply]
Ah, I see what you mean now by:
> Windows key to the left or right Command key
> just be able to specify right vs left command

Pretty sure this is not a feature right now, unfortunately. You're welcome to open a new feature request though.

**Comment by timguy99** [comment]
How does one do a simple mapping of Windows ALT key -> MAC COMMAND key from the server side in Windows? I can't even figure out the interface in the hotkeys section. It prompts for keypresses continuously. Also if I try to map Windows Start key it just pops up the start menu and doesn't recognized the keypress in the hotkeys interface?

Am I trying to do this incorrectly? Do I have to edit a config file somewhere?

**Reply by ceremona** [reply]
I'm using a linux server and mac client and have the same frustrating question.

**Comment by sithlord48** [comment]
In the Computers tab of the server's configuration dialog. Double click on the screen you want to modify  this is not done in the hotkeys section

**Reply by timguy99** [reply]
All, thanks for the quick reply. I see that now. HOWEVER. What does META and SUPER mean in MAC terms? I'm looking for COMMAND and OPTION which don't seem to exist.

**Reply by sithlord48** [reply]
Meta / super is the apple button iirc.

**Reply by sithlord48** [reply]
IIRC on mac its 

control = ctrl
alt = option 
meta = command

**Reply by timguy99** [reply]
Thanks again on the overall info. I just messed around by trial and error and figured things out.

I think what it is, is SUPER == MAC COMMAND / WINDOWS (or START) , ALT == MAC OPTION (also ALT). I'm not sure what META is on either side. I left META mapping to META. Maybe its the FN key on both Windows and MAC. Haven't tried that as I don't use the function key on MAC all that much or at least not regularly for anything I can think of right now. Will try that some other time.
